### PR TITLE
issue - 34

### PR DESCRIPTION
**Pull request by dpolozov**
resizing video player using css at specific screen sizes.
added player controls for ability to go full screen.

### CODE REVIEW

File: src/components/CampaignPage.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ import React, {lazy, useState, Component} from 'react';
...
 import { Container, Row, Col, Card, ProgressBar, Button, Modal, Image, InputGroup, FormControl } from 'react-bootstrap';
-import { ChevronLeft, Gift, CheckCircle, ExclamationTriangle, HourglassSplit, XCircle } from 'react-bootstrap-icons';
+import { ChevronLeft, Gift, CheckCircle, ExclamationTriangle, HourglassSplit, XCircle, Fullscreen} from 'react-bootstrap-icons';
 import ReactPlayer from 'react-player';
 import { Link } from "react-router-dom";
 import { Trans } from 'react-i18next';
 import i18n from '../util/i18n';
 import '../css/campaignPage.css';
 import { Editor, EditorState, convertFromRaw } from "draft-js";
+import screenfull from 'screenfull';
```
Comment (grishick) [inline src/components/CampaignPage.js]: are these two imports being used here?

File: src/components/CampaignPage.js [hunk 2]
```diff
@@ -2,13 +2,15 @@ import React, {lazy, useState, Component} from 'react';
 import config from "react-global-configuration";
 import axios from 'axios';
 import { Container, Row, Col, Card, ProgressBar, Button, Modal, Image, InputGroup, FormControl } from 'react-bootstrap';
-import { ChevronLeft, Gift, CheckCircle, ExclamationTriangle, HourglassSplit, XCircle } from 'react-bootstrap-icons';
+import { ChevronLeft, Gift, CheckCircle, ExclamationTriangle, HourglassSplit, XCircle, Fullscreen} from 'react-bootstrap-icons';
```
Comment (grishick) [inline src/components/CampaignPage.js]: I don't think Fullscreen icon is being used in the current solution.

**Review by grishick — CHANGES_REQUESTED** [review]
Looks like you left behind experimental changes

File: package.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "^4.0.1",
     "react-uuid": "^1.0.2",
+    "screenfull": "^5.1.0",
```
Comment (grishick) [inline package.json]: do we need this?

**Review by grishick — CHANGES_REQUESTED** [review]
I think you missed one more

**Comment by dpolozov** [comment]
I thought I did uninstall the screenful. But didn't check if it got
removed. Out of the house right now

**Review by grishick — APPROVED** [review]

**Pull request merged**